docs/Getting-Started-with-Balance-Ball.md

```diff
@@ -249,25 +249,37 @@ example algorithm for use with ML-Agents. For more information on PPO,
 OpenAI has a recent [blog post](https://blog.openai.com/openai-baselines-ppo/) 
 explaining it.
 
-In order to train the agents within the Ball Balance environment:
 
-1. Open `python/PPO.ipynb` notebook from Jupyter.
-2. Set `env_name` to the name of your environment file earlier.
-3. (optional) In order to get the best results quickly, set `max_steps` to 
-50000, set `buffer_size` to 5000, and set `batch_size` to 512.  For this 
-exercise, this will train the model in approximately ~5-10 minutes.
-4. (optional) Set `run_path` directory to your choice. When using TensorBoard 
-to observe the training statistics, it helps to set this to a sequential value 
+To train the agents within the Ball Balance environment, we will be using the python 
+package. We have provided a convenient python wrapper script called `learn.py` which can be passed 
```

Comment:
"which can be passed arguments which are used to configure the training" --> "which accepts arguments used to configure the training phase"